extensions/wikia/Email/Controller/CommentController.class.php

```diff
@@ -104,7 +104,7 @@ protected function getSalutation() {
 	}
 
 	protected function getSummary() {
-		$articleTitle = $this->title->getText();
+		$articleTitle = $this->pageTtitle->getText();
 
 		return wfMessage( $this->getSummaryKey(), $articleTitle )
 			->inLanguage( $this->targetLang )
```

Comment:
Like Damian said above - the reason of the bug was the $title property name, and now it is fixed.
Since this line was not causing the problem, I'm gonna leave it like it is.